packages/reports/addon/templates/components/metric-selector.hbs

```diff
@@ -6,7 +6,7 @@
     @selected={{this.selectedMetrics}}
     @title="Metrics"
     @contentClass="navi-list-selector__content--metric"
-    class="navi-list-selector--request-preview"
+    class="navi-list-selector"
```

Comment:
```suggestion
```
We should be able to get rid of this now since `<NaviListSelector />` should already apply the class